try {
        
      //   if (arr[(i - j)] < 0) {
      //     throw 'error'
      //   }
      // } catch {
      //   console.log('error')
      //   error = true
      // }